src/graph.rs

```diff
@@ -693,7 +694,9 @@ pub struct WorkspaceMember {
 #[serde(rename_all = "camelCase")]
 #[serde(tag = "kind")]
 pub enum Module {
-  Esm(EsModule),
+  // todo(#239): remove this when updating the --json output for 2.0
+  #[serde(rename = "esm")]
+  Js(JsModule),
```

Comment:
I found it simplifies the downstream code to use `Module::Js` here and then we can introduce `Module::Wasm` later. The benefit is that it forces people to handle Wasm modules specifically and also if someone has a `JsModule` then they don't need to worry about if they can get a string out of it or not.

Couldn't think of a better name because `ScriptModule` has the connotations that it's not an ES module.

Reply:
Current name is fine 👍